core/src/main/java/com/klaytn/caver/Caver.java

```diff
@@ -32,15 +31,15 @@
     public static String DEFAULT_URL = "http://localhost:8551";
 
     /**
-     * @deprecated Please use <code>caver.rpc.klay</code> instead.
      * @see RPC#klay
+     * @deprecated Please use <code>caver.rpc.klay</code> instead.
```

Comment:
If the changes are not essential, please keep the existing code.

Reply:
@jimni1222 

Yeap. This is not essential though.
It looks better not to change it.

I will reflect your feedbacks and revert it.

Reply:
I revert it with [Revert code formatting and import optimization for Caver](https://github.com/klaytn/caver-java/pull/215/commits/d45909eb60a5cfa78cd45b9f0489cc00624f6b9b)